packages/rest-typings/src/v1/users/UsersListStatusParamsGET.ts

```diff
@@ -0,0 +1,44 @@
+import Ajv from 'ajv';
+
+import type { PaginatedRequest } from '../../helpers/PaginatedRequest';
+
+const ajv = new Ajv({
+	coerceTypes: true,
+});
+
+export type UsersListStatusParamsGET = {
+	status: 'active' | 'all' | 'deactivated' | 'pending';
+	roles: string[];
+	searchTerm: string;
+} & PaginatedRequest;
```

Comment:
PaginatedRequest is a generic type, you can put `PaginatedRequest<YourType>`. It works almost the same, but...